oceanful/polyclip-go
# Improve normalization of test inputs and add a test for akavel#8.

Test improvements:
* Ensure that paths being compared in tests have a consistent orientation.
* Remove zero-length segments from inputs (e.g. when the first and last Point are redundant).

Finally, verify that https://github.com/akavel/polyclip-go/issues/8 is fixed.
